src/getProducts.js

```diff
@@ -0,0 +1,18 @@
+import axios from 'axios';
+
+const getProducts = async () => {
+  let dataFetched;
+
+  try {
+    const headers = { ' Access-Control-Allow-Origin': '*' };
+    const data = await axios.get('https://fakestoreapi.com/products?limit=9', headers);
+    localStorage.setItem('products', JSON.stringify(data.data));
+    dataFetched = data.data;
+  } catch (err) {
+    console.log(err);
```

Comment:
- [ ] good job on changing the suggestions the previous CR made 👍 however in the previous review was also mentioned to remove this `console.log` and the CR also explains why you should do so, please remove all the console logs in your file.

Reply:
Hello @mikethreels,

We did make these changes. Kindly check this [commit](https://github.com/blessedjasonmwanza/ecommerce-rating/pull/36/commits/00038fa793687eaba32dc7401868c05568ef5905)